data/common.yaml

```diff
@@ -159,3 +159,7 @@ icingaweb2::module::puppetdb::certificates: {}
 icingaweb2::module::perfdatagraphs::ensure: present
 icingaweb2::module::perfdatagraphs::git_repository: https://github.com/NETWAYS/icingaweb2-module-perfdatagraphs.git
 icingaweb2::module::perfdatagraphs::install_method: git
+
+icingaweb2::module::perfdatagraphsgraphite::ensure: present
```

Comment:
why is some data in common.yaml but others are directly in the puppet class? IMO it makes more sense to have all of them in the puppet class, because that enables puppet-strings to pick it up and document it in the REFERENCE.md.

Reply:
Will be changed with v6.0.0 so that only default values that are not static will be stored in data. Maybe you remember there to data and back because of problems with documentation via puppet strings.

Reply:
Ok. Then I'll start rebuilding it now. This will also affect the other 3 modules. And really all static parameter values?

Is it ok, for example, to specify different installation methods via data in module for the icingaweb modules depending on the platform?

Reply:
yes & yes